android-quick-start/README.md

```diff
@@ -0,0 +1,88 @@
+# Quickstart: Create an Android app with Amazon Location Maps and Places API
+
+This quickstart app creates a quick start application that demonstrates the following:
+
+1. Adding Amazon Location Maps using Esri as data source to the app.
+2. Adding reverse geocoding using Amazon Location Places API to demonstrate a place picker.
+
+After a successful build the app will look like:
+
+![](screenshot.png)
+
+## Prerequisities
+1. A Amazon Location map resource
+1. A Amazon Location place-index resource
+1. A Cognito identity pool for unauthorized access
+
+## How to set up the project
+Before the project can be built we need the following steps to be completed:
+1. Configuring amplify to set map, place-index, and cognito identity-pool id.
+1. Setting up Android SDK path using `local.properties` file or opening the project in Android Studio.
+
+### Configuring Amplify
+This project uses a JSON configuration following Amplify's configuration schema in `app/res/raw/amplifyconfiguration.json`. To use the quick start, we need to replace the following in the configuration JSON.
```

Comment:
```suggestion
This project uses a JSON configuration following Amplify's configuration schema in `app/src/main/res/raw/amplifyconfiguration.json`. To use the quick start, we need to replace the following in the configuration JSON.
```

Reply:
Ah, thank you for catching that. I will take this suggestion.